readme.md

```diff
@@ -8,7 +8,7 @@ For the following quiz, there will be a series of questions or directions follow
 What is Git, what problem does it solve?
 
 ```
-answer goes here
+Git is a version control system. It is used to solve the problem of tracking changes among many collaborators.
 
 ```
 
```

Comment:
👍  It does do mainly that, but it can help you keep track of your own versions as well.